### PR TITLE
Rating and Modal Added

### PR DESCRIPTION
**Pull request by ashifkhn**
Rating and Modal Added in the Library.
This project isnt mobile responsive yet. 

Inspect the deploy log: https://app.netlify.com/sites/whyui/deploys/62091705415b5a00075a7111
Browse the preview: https://deploy-preview-11--whyui.netlify.app/

### CODE REVIEW

File: components/Modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+				<a href="../Rating/rating.html" class="navbarItem">Rating</a>
+				<a href="../Text Utilities/textUtilities.html" class="navbarItem"
+					>Text Utilities</a
+				>
+			</div>
+
+			<div class="main">
+				<h1 class="textTitle">Modal</h1>
+				<p class="textDetails">
+				Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ipsa placeat praesentium fugit accusamus vel eos perspiciatis ipsam fugiat! In, dignissimos?
```
Comment (harshitpaliwal95) [inline components/Modal/modal.html:59]: change description

**Review by harshitpaliwal95 — APPROVED** [review]
rating looks good

**Review by harshitpaliwal95 — CHANGES_REQUESTED** [review]
need to fix modal 
https://u-and-i.netlify.app/docs.html#Modal check for refrence


Screenshot 2022-02-13 at 10 24 28 PM

**Review by vikrantkode — COMMENTED** [review]
In UI , modal can be better. you can prefer the reference given.

**Pull request merged**